grakn-graql/src/main/java/ai/grakn/graql/internal/hal/HALUtils.java

```diff
@@ -58,9 +58,9 @@
     final static String SUB_EDGE = "sub";
```

Comment:
This can also be `Schema.EdgeLabel.SUB.getLabel()` same for `ISA_EDGE` which is not shown in this diff